local-volume/provisioner/CHANGELOG.md

```diff
@@ -1,9 +1,20 @@
+# v2.0.0
+This version is incompatible and has breaking changes with v1!
+* Remove default config, a configmap is now required.
+* Configmap data is changed from json to yaml syntax.
+* All local volumes must be mount points.  For directory-based volumes, a
+  bind-mount must be done in order for the provisioner to discover them.
+* Detected capacity is rounded down to the nearest GB.
+* Mount propagation is required to discover new mount points after the
```

Comment:
Does it make sense to refer to https://kubernetes.io/docs/concepts/storage/volumes/#mount-propagation or mention k8s 1.8 has it?

Reply:
Done.  The top level README contains specific instructions on how to enable mount propagation and in what version